generals/gui/properties.py

```diff
@@ -59,8 +53,9 @@ def game_speed(self):
         return self.__game_speed
 
     @game_speed.setter
-    def game_speed(self, value: int):
-        self.__game_speed = value
+    def game_speed(self, value: float):
+        new_speed = min(32.0, max(0.25, value))  # clip speed
```

Comment:
Personally, I would enjoy even higher speeeed, something like 128.

Reply:
`32.0` means 64fps (and also 64 moves per second) -> monitors with 60Hz will render the game as if it is skipping moves

Reply:
qucik maths

Reply:
Sometimes getting faster to the end or quickly getting overview about the game is more important than being able to see/render every move.

Reply:
Then maybe a better thing to do would be to add a button for skipping to the end (similar to `r` for restart)?
I think replaying faster than we can see is not that good of an idea.

Reply:
feature request: create slider in the right panel where you can skip over the game :D

Reply:
I will submit an issue on this.

Reply:
https://github.com/strakam/Generals-RL/issues/73